src/sentry/relay/config/__init__.py

```diff
@@ -41,6 +40,7 @@
 from sentry.utils.http import get_origins
 from sentry.utils.options import sample_modulo
 
+from ...dynamic_sampling.utils import is_on_dynamic_sampling
```

Comment:
you should use full import path. E.g. sentry.dynamic_sampling.utils

Reply:
I think Pycharm did this, will fix! Thanks